src/components/loadflow-result.js

```diff
@@ -137,9 +138,48 @@ const LoadFlowResult = ({ result, studyUuid, nodeUuid }) => {
             </TableCell>
         );
     }
+    const loadFlowConstraintscolumns = [
```

Comment:
useMemo maybe, because we have translation ?

Reply:
Done